doc/interfaces/denoise.rst

```diff
@@ -0,0 +1,90 @@
+=========
+Denoising
+=========
+
+This tutorial walks through the steps for denoising of diffusion-weighted MR images.
+There can be multiple denoising techniques.
+
+First you need to download the desired data.
+
+------------------------
+Downloading the Data Set
+------------------------
+
+You can view all the datasets using the command::
+
+    dipy_fetch list
+
+To fetch some specific dataset, you can use the following command::
+
+    dipy_fetch specific_dataset
+
+For example, to download sherbrooke_3shell dataset, you can use the following
+command::
+
+    dipy_fetch sherbrooke_3shell
+
+-------------------------
+Denoising using Local PCA
+-------------------------
+
+PCA-based denoising algorithms are effective denoising methods because they explore 
+the redundancy of the multi-dimensional information of diffusion-weighted datasets
+
+We are using isbi2013_2shell dataset. You can also use your own data.
```

Comment:
Use double backticks for verbatim/code parts: (double_backticks)isbi2013_2shell(double_backticks).

Note that I am using (double_backticks) to avoid Markdown predating the actual double backticks.

Reply:
Addressed in d49dc04